src/components/results/PdfSimulation.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div ref="document" class="pdf-container">
+    <button
+      class="bg-black text-white rounded-lg py-4 px-3 flex flex-row gap-3 w-fit items-center ml-auto"
+      @click="exportToPDF()"
+      :disabled="isLoading"
+    >
+      <img :src="iconDownload" alt="" />
+      <span class="font-dm-sans font-medium text-base">
+        Télécharger en PDF
+      </span>
+      <UiSpinnerLoading v-if="isLoading" width="20" height="20" />
```

Comment:
width and height are the same as the default value, perhaps no need to set it here.